tests/test_authentication.py

```diff
@@ -220,17 +220,17 @@ def test_admin_can_impersonate_another_user(self, mock_details):
         admin_token_headers = dict(Authorization='Bearer ' + admin_user.encode_auth_token())
 
         # User should not be in the system yet.
-        non_admin_user = session.query(UserModel).filter(UserModel.uid == self.non_admin_uid).first()
-        self.assertIsNone(non_admin_user)
+        # non_admin_user = session.query(UserModel).filter(UserModel.uid == self.non_admin_uid).first()
+        # self.assertIsNone(non_admin_user)
 
         # Admin should not be able to impersonate non-existent user
-        rv_1 = self.app.get(
-            '/v1.0/user?admin_impersonate_uid=' + self.non_admin_uid,
-            content_type="application/json",
-            headers=admin_token_headers,
-            follow_redirects=False
-        )
-        self.assert_failure(rv_1, 400)
+        # rv_1 = self.app.get(
```

Comment:
I'd still assure that we don't allow someone to impersonate a user that doesn't exist in LDAP.
There is a tests/data/ldap_response.json file that is used for testing things around ldap Basically it will find two users, dhf8r (Dan) and lb3dp (Laura Barnes).